warehouse/templates/includes/accounts/edit-profile-button.html

```diff
@@ -14,4 +14,16 @@
 
 {% if request.user == user %}
   <a href="{{ request.route_path('manage.account') }}" class="button button--primary author-profile__edit-button">Edit profile</a>
+
+  <div class="sidebar-section">
+    <div class="sidebar-section__spacer"></div>
```

Comment:
Hi @coreygirard, thanks for this PR !

I think we can avoid adding an empty div here, instead we can add a modifier to `sidebar-section`.

e.g. `sidebar-section sidebar-section--spaced`. 

Then we can add a `margin-top: 20px` style in the css. 

Typically, we add modifiers at the bottom of the codeblock like this `&--spaced`, and add a small comment in the documentation at the top of the page.  In this case, there seems to also be some out of date comments there too.

The button group CSS file is a good place to look for a good example :)

Let me know if this isn't clear, or if you'd like me to make these changes for you :)

Reply:
So I had originally tried to do basically exactly that, but for some reason, it didn't end up showing the space. I just tried again, with no luck. But my CSS skills are quite limited, so if you wouldn't mind making the edit, that would be much appreciated :)